studybuddies/src/app/services/httperror-interceptor.service.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable, OnInit } from '@angular/core';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7202842)
```suggestion
import { Injectable, OnInit } from "@angular/core";
```